policy/modules/services/dbus.if

```diff
@@ -108,6 +135,21 @@ template(`dbus_role_template',`
 		dontaudit $3 $1_dbusd_t:netlink_selinux_socket { read write };
 	')
 
+	optional_policy(`
+		gen_require(`
+			type $1_systemd_t;
+		')
+
+		dbus_spec_domtrans($1_systemd_t, $1_dbusd_t)
+		allow $1_dbusd_t $1_systemd_t:unix_stream_socket rw_socket_perms;
+
+		allow $1_systemd_t $1_dbusd_t:process signal_perms;
+		allow $1_systemd_t $1_dbusd_t:unix_stream_socket server_stream_socket_perms;
+
+		allow $1_systemd_t $1_dbusd_t:dir search_dir_perms;
+		allow $1_systemd_t $1_dbusd_t:file read_file_perms;
+	')
```

Comment:
You are not allowed to reference $1_systemd_t here.

dbusd is just a "systemd_user_spec_daemon_domain($1, $1_dbusd_t, dbusd_exec_t)" i guess. Should look into implementing something like that